recipe/conda_build_config.yaml

```diff
@@ -7,9 +7,11 @@
 # Select the MACOSX Deployment target
 # This will change depending on the version of go
 # For the correct deployment target, please see go's Release notes
-# c.f.: https://golang.org/doc/go1.16#darwin
+# c.f.: https://golang.org/doc/go1.18#darwin
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - '10.12'                # [osx and x86_64]
+  - '10.13'                # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - '10.14'                # [osx and x86_64]
```

Comment:
Looking at build logs xcode 12 is being used and this is problematic because:

[```Xcode 12 requires an Intel-based Mac running macOS Catalina 10.15.4 or later.``` ](https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes)

So change to 10.15.4?

Reply:
@xhochy assuming it is possible, do you mind if I push some commits that branch?

Reply:
No, please don't. This variable only affects the SDK but not the OS version. The issue is probably somewhere else.